package.json

```diff
@@ -77,19 +77,19 @@
     "cookie-parser": "1.4.5",
     "core-js": "3.6.5",
     "date-fns": "2.14.0",
-    "debug": "4.2.0",
```

Comment:
This looks like it's going the other way? Guessing it might have been a rebase conflict or something?

Reply:
Hmmm - look at the versions on https://www.npmjs.com/package/debug.

Anyhow, I've installed debug@4.2.0 rather than debug@latest and all is well...